### PR TITLE
Feature/styleguide styling

### PR DESCRIPTION
**Pull request by claudio147**
## Pull request
*Styleguide styling => the h1, h2 from typography section can be ignored, will be fixed with another branch (Feature/wf 441 fonts)*
 
### Ticket
*https://movento.atlassian.net/browse/VFI-15*
 
### Browser testing
- *http://localhost:6060/styleguide*
 
### Checklist
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [x] Did run automated tests and linters
- [x] Did review code
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Re-assign ticket to developer

### CODE REVIEW

File: app/setup/styleguide.styles.js [hunk 1]
```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
```
Comment (patric-eberle) [inline app/setup/styleguide.styles.js]: - [ ] I would keep this to the left. Else you could miss the titles on wide screens.

File: app/setup/styleguide.styles.js [hunk 2]
```diff
@@ -32,6 +34,28 @@ module.exports = {
...
+      },
+      heading2: {
+        'backgroundColor': '#2c68a0',
+        'color': '#fff',
+        'width': 'calc(100% + 32px)',
+        'padding': '0px 15px',
```
Comment (patric-eberle) [inline app/setup/styleguide.styles.js]: - [ ] Can you align the title with the main content? Currently its a bit more to the left.

File: app/setup/styleguide.styles.js [hunk 3]
```diff
@@ -32,6 +34,28 @@ module.exports = {
...
+        '& a': {
+          '&:hover': {
+            textDecoration: 'none',
+            color: '#fff !important',
+            cursor: 'pointer !important'
+          }
+        }
+      }
+    },
     ComponentsList: {
```
Comment (patric-eberle) [inline app/setup/styleguide.styles.js]: - [ ] Can you increase the `margin-bottom` for the sections (the one with blue h2)?

File: app/styleguide/components/s-welcome.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+### Welcome to the component styleguide
+Here you see all the project components.
+
+For example pages go to
+<a href="http://localhost:8080/styleguide" target="_blank">localhost:8080/styleguide</a><br>
+dev-server have to run:
```
Comment (patric-eberle) [inline app/styleguide/components/s-welcome.md]: - [ ] Use `To serve the Vue environment run:` or similar.

File: app/setup/styleguide.styles.js [hunk 5]
```diff
@@ -32,6 +34,28 @@ module.exports = {
...
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
+      },
+      heading2: {
+        'backgroundColor': '#2c68a0',
+        'color': '#fff',
+        'width': 'calc(100% + 32px)',
+        'padding': '0px 15px',
+        'marginLeft': '-33px',
```
Comment (patric-eberle) [inline app/setup/styleguide.styles.js]: - [ ] This is causing problems when the navigation disapears. Maybe you can extend the background by adding a pseudo element and remove the negative margin? There is a `rsg--hasSidebar-2` on a parent element if the sidebar is visible. Maybe this also helps.

File: app/setup/styleguide.styles.js [hunk 6]
```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
```
Comment (patric-eberle) [inline app/setup/styleguide.styles.js]: - [ ] The `:hover` is impacted by the global styles of the project. Would be nice if they are custom as well (just underline, also an issue on the zoom icons).

File: app/setup/styleguide.styles.js (hunk 5 above)
Comment (claudio147) [reply app/setup/styleguide.styles.js]: The problem doesn't occur anymore because of the changed subtitle alignment

**Comment by claudio147** [comment]
@patric-eberle Fixed the issues - please check again (specially where I added a custom styleguide SCSS file, maybe there is a better place for it?)

**Pull request merged**